Assets/js/info.js

```diff
@@ -89,8 +115,8 @@ var randomMovie = function() {
     .then(response => response.json())
         .then(data => {
             console.log(data);
-            localStorage.setItem("movieObject", JSON.stringify(data));
-            localStorage.setItem("testID", movieId)
+            storageBox = [data, movieID] //created this variable to match array style of normal movieObject
```

Comment:
We already have a random movie function in app.js. Do we need the same function here or can it be removed?

Reply:
The basic idea of what I'm thinking (and others may have been thinking along the same lines and we just never talked about it), was that instead of sending the object of the movie clicked through local, we only send it's _ID_, then we run another fetch request on the info page and populate it with the new one. Henry inserted a fetch request to get a movie by a random idea on that page for the randomizer, and it would simplify the code for sure if I just slapped one on there and modified the existing code to display the new data. Additionally, when you call for a movie by ID, it brings you more information about it than you get in the search object from genre, so it will be easier to get some of the info I was struggling to get passed to the next page.